docs/modules/ROOT/pages/how-tos/testing/integration/accessing-private-repositories.adoc

```diff
@@ -0,0 +1,71 @@
+= Access Pipelines and Tasks in Private Repositories
+
+Integration Test Scenarios point to their corresponding pipeline with a link:https://tekton.dev/docs/pipelines/git-resolver/[Git Resolver]. The Git Resolver schema is inherited from Tekton and functions in the same way. Users should create an access token for their repository in link:https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens[Github] or link:https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html[Gitlab] then link:../configuring/creating-secrets.adoc#creating-source-control-secrets[store that token in a secret] in their namespace.
+
+Once this is done, the Resolver should be updated with the fields `token` and `tokenKey`, which provide the name of the secret and the key within the `data` section of that secret in which the access token is stored.
```

Comment:
This _feels_ like the wrong place to put information about creating what is effectively a source control secret. Do we have the steps anywhere else? Do you think that this is this a new type of secret that we should document creating on https://konflux-ci.dev/docs/how-tos/configuring/creating-secrets/?

Reply:
I don't feel strongly enough about it to change now. We can always iterate.